### PR TITLE
PBHEntryPoint Uninit Tests

### PR DESCRIPTION
**Pull request by 0xForerunner**

### CODE REVIEW

File: contracts/test/TestSetup.sol [hunk 1]
```diff
@@ -61,8 +61,8 @@ contract TestSetup is Test {
         vm.deal(address(this), type(uint128).max);
         vm.deal(address(safe), type(uint256).max);
 
-        // Deposit some funds into the Entry Point from the Mock Account.
-        entryPoint.depositTo{value: 10 ether}(address(safe));
+        // TODO: Deposit some funds into the Entry Point from the Mock Account.
+        // entryPoint.depositTo{value: 10 ether}(address(safe));
```
Comment (0xOsiris) [inline contracts/test/TestSetup.sol]: Can we uncomment this? This is necessary for `testFullFlow` Setup in the Signature aggregation tests.

**Review by 0xOsiris — APPROVED** [review]
Looks good, just one change!

File: contracts/test/TestSetup.sol (hunk 1 above)
Comment (0xForerunner) [reply contracts/test/TestSetup.sol]: It wasn't working when I ran it, unless you fixed it on main. Lemme check.

Comment (0xOsiris) [reply contracts/test/TestSetup.sol]: You need to fork mainnet when running the tests. `forge test -f $ETHEREUM_PROVIDER`. As we are just wrapping the `entryPoint` around the mainnet address. It's passing in CI on main I believe

**Pull request merged**